### ISSUE TITLE
Reservation feature missing

### ISSUE THREAD

**Issue by ammar88** [ammar88/Vcafe#3]
The book a table wording is missing from the left of the header, when the user clicks this the header should drop down showing a reservations section see page 5 of the attached presentation.

**Comment by ammar88** [comment]
Feature added, waiting on real reservations plug-in.

**Comment by ammar88** [comment]
Adding plugin will be new issue...
